geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/stats/package.scala

```diff
@@ -0,0 +1,19 @@
+/***********************************************************************
+ * Copyright (c) 2013-2019 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.index
+
+package object stats {
+
+  /**
+    * Marker trait for classes with stats
+    */
+  trait HasGeoMesaStats {
```

Comment:
moved this class to a package object instead of having multiple top-level classes in a single file